aim/api/resource.py

```diff
@@ -86,7 +89,6 @@ class Tenant(AciResourceBase):
 
     identity_attributes = ['name']
     other_attributes = ['display_name']
-    db_attributes = []
 
     _aci_mo_name = 'fvTenant'
     _tree_parent = None
```

Comment:
If Tenant name becomes a foreign Key then a Tenant resource will be mandatory, right? Will it be created implicitly?